auto_tests/encryptsave_test.c

```diff
@@ -207,12 +207,12 @@ static Suite *encryptsave_suite(void)
 int main(void)
 {
     setvbuf(stdout, nullptr, _IONBF, 0);
-    srand((unsigned int) time(nullptr));
+    srand((uint32_t) time(nullptr));
 
     Suite *encryptsave =  encryptsave_suite();
     SRunner *test_runner = srunner_create(encryptsave);
 
-    int number_failed = 0;
+    uint8_t number_failed = 0;
```

Comment:
Maybe just get rid of the SRunner stuff. That's just old compat stuff. Now we just call the test function directly (and we shouldn't use START_TEST anymore, just declare a static function manually).

Reply:
I was never quite sure what SRunner does tbqh. Is my HEAD now equivalent?